### PR TITLE
Excel properties

### PR DESCRIPTION
**Pull request by scraperdragon**

### CODE REVIEW

File: messytables/excel.py [hunk 1]
```diff
@@ -22,32 +24,47 @@
...
-
 class XLSTableSet(TableSet):
     """An excel workbook wrapper object.
     """
 
     def __init__(self, fileobj=None, filename=None,
-                 window=None, encoding=None):
+                 window=None, encoding=None, with_formatting_info=True):
+        def get_workbook():
+            print filename, len(read_obj)
```
Comment (drj11) [inline messytables/excel.py]: is this a debug print?

File: messytables/excel.py [hunk 2]
```diff
@@ -22,32 +24,47 @@
...
 
-
 class XLSTableSet(TableSet):
     """An excel workbook wrapper object.
     """
 
     def __init__(self, fileobj=None, filename=None,
-                 window=None, encoding=None):
+                 window=None, encoding=None, with_formatting_info=True):
+        def get_workbook():
```
Comment (drj11) [inline messytables/excel.py]: i believe this function definitioin should go after the docstring (that starts on line 44, below).

File: messytables/excel.py [hunk 3]
```diff
@@ -22,32 +24,47 @@
...
-
 class XLSTableSet(TableSet):
     """An excel workbook wrapper object.
     """
 
     def __init__(self, fileobj=None, filename=None,
-                 window=None, encoding=None):
-        '''Initilize the tableset.
+                 window=None, encoding=None, with_formatting_info=True):
+        '''Initialize the tableset.
```
Comment (scraperdragon) [inline messytables/excel.py]: Explain why we're doing this.

File: messytables/excel.py [hunk 4]
```diff
@@ -73,15 +90,79 @@ def raw(self, sample=False):
...
+            value = datetime(year, month, day, hour,
+                             minute, second)
+        messy_cell = XLSCell(value, type=cell_type)
+        messy_cell.sheet = sheet
+        messy_cell.xlrd_cell = xlrd_cell
+        return messy_cell
+
+    @property
+    def topleft(self):
+        # TODO
```
Comment (scraperdragon) [inline messytables/excel.py]: make trello card for spanning cells

File: messytables/excel.py [hunk 5]
```diff
@@ -73,15 +90,79 @@ def raw(self, sample=False):
...
+    def __init__(self, cell):
+        self.xf = cell.sheet.book.xf_list[cell.xlrd_cell.xf_index]
+        self.font = cell.sheet.book.font_list[self.xf.font_index]
+
+    def get_bold(self):
+        return self.font.weight > 500
+
+    def get_size(self):
+        """in pixels"""
+        return self.font.height / 20
```
Comment (scraperdragon) [inline messytables/excel.py]: yuck, int division

**Pull request merged**